src/Stack/Types/TemplateName.hs

```diff
@@ -98,6 +112,11 @@ mkTemplateName s =
                           AbsPath (Path fp) -> [|AbsPath (Path fp)|]
                           RelPath (Path fp) -> [|RelPath (Path fp)|]
                           UrlPath fp -> [|UrlPath fp|]
+                          RepoPath (RepoTemplatePath sv u t) ->
+                            case sv of
+                                Github    -> [|RepoPath $ RepoTemplatePath Github u t|]
+                                Gitlab    -> [|RepoPath $ RepoTemplatePath Gitlab u t|]
+                                Bitbucket -> [|RepoPath $ RepoTemplatePath Bitbucket u t|]
```

Comment:
I was wondering about this part, since I'm not clear about what it does exactly. 

Could it do without the pattern match and just be:

```
RepoPath (RepoTemplatePath sv u t) -> [|RepoPath $ RepoTemplatePath sv u t|]
```

Reply:
No, that would cause TemplateHaskell to complain that there is no instance of something-or-other class. (`Rep`?) The alternative is to define an instance of that class, but Michael's recommendation was that it wasn't worth it.